### PR TITLE
CNV-32389: Remove delete msg from nic modal if vm is running

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

> Add a brief description

## 🎥 Demo

Remove message in nic modal if VM is not running

After:
image

Before:
image

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx [hunk 1]
```diff
@@ -87,14 +88,16 @@ const NetworkInterfaceActions: FC<NetworkInterfaceActionsProps> = ({
...
-              'Deleting a network interface is supported only on VirtualMachines that were created in versions greater than 4.13  or for network interfaces that were added to the VirtualMachine in these versions.',
-            )}
-            component={'h6'}
-            isInline
-            variant={AlertVariant.warning}
-          />
+          {isVMRunning(vm) && (
+            <Alert
+              title={t(
+                'Deleting a network interface is supported only on VirtualMachines that were created in versions greater than 4.13  or for network interfaces that were added to the VirtualMachine in these versions.',
```
Comment (hstastna) [inline src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx:94]: I can see one extra space between "4.13" and "or" ...

File: src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx [hunk 2]
```diff
@@ -23,6 +23,7 @@ import {
   KebabToggle,
 } from '@patternfly/react-core';
 import { removeInterface } from '@virtualmachines/actions/actions';
+import { isVMRunning } from '@virtualmachines/utils';
```
Comment (hstastna) [inline src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx:26]: ERROR in src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx:26:10
TS2305: Module '"@virtualmachines/utils"' has no exported member 'isVMRunning'.

Comment (metalice) [reply src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx:26]: dint know why it didnt commit the second file, thanks

**Review by avivtur — APPROVED** [review]

**Pull request merged**